test/bondingCurve.test.js

```diff
@@ -94,7 +94,16 @@ describe('Bonding curve', function () {
     let balanceB = initialBalanceB
 
     await tokenB.connect(accountB).approve(bondingCurve.address, balanceB)
+
+    // Buying
     for (let i = 1; i <= tokenASupply; i++) {
+
+      var buyAmount = 1
```

Comment:
Here and below: not sure what the test is accomplishing, maybe have `buyAmount = 2` or `= i` instead?

Reply:
checking if total price is fine